### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zedgamer9128**
Changed the required things and also some things i think should be changed if you don't like it please change that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,11 @@ EcoHabit is a web app project that aims to create awareness of the environment a
 
 It will visualize users' habits by tracking their activities (recycling, commuting, eating, etc.) and encourage them to improve their habits with better alternative options.
 
-It also targets to be users' go-to place to find the closest recycling locations and give them clear instructions about how to divide and recycle each material.
-
+It also aims to become users' preferred destination for locating the nearest recycling centers while providing them with concise instructions on how to sort and recycle various materials.
```
Comment (lugenx) [inline README.md:25]: Let's revert this line back to the original version, rest looks good :+1:

**Review by lugenx — CHANGES_REQUESTED** [review]
just one comment to address for you, other than that, this looks great, thank you for catching additional issues. @zedgamer9128

**Comment by zedgamer9128** [comment]
Glad to help

**Comment by zedgamer9128** [comment]
If you have more such type of issues you can assign me

**Comment by lugenx** [comment]
> If you have more such type of issues you can assign me

Of course, make sure keep eye on the issues page, and comment under any issue if you think you can solve. 

before merging this PR, please address the comment I have above: 
https://github.com/lugenx/ecohabit/pull/254#discussion_r1370781457

**Comment by zedgamer9128** [comment]
@lugenx should be good to go now

**Review by lugenx — APPROVED** [review]
great, thank you!

**Pull request merged**